.github/workflows/create-release.yml

```diff
@@ -16,15 +16,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Checkout spinalcordtoolbox
+    - name: Checkout spinalcordtoolbox (main branch)
       uses: actions/checkout@v2
       with:
-        ref: ${{ env.RELEASE_BRANCH }}
+        ref: ${{ env.MAIN_BRANCH }}
+        fetch-depth: 0
```

Comment:
NB: `fetch-depth: 0` is needed for the merge from master into release. 

Without it, we get a "fatal: refusing to merge unrelated histories" error, since [`actions/checkout@v2`'s default fetch depth is `1`](https://github.com/actions/checkout/issues/125) (which is too small to know the point where the two branches have shared histories).